site/src/templates/componentPage.tsx

```diff
@@ -44,7 +44,7 @@ export default ({ data, pageContext, location }) => {
   )
 
   const renderStorybookIFrame = () => {
-    if (!md.frontmatter.demoStoryId && md.frontmatter.title !== "Overview") {
```

Comment:
There was a logic bug: the Overview does not have a demoStoryId, so it was not matching this line. I moved the logic for the overview outside the component, and I don't think this warning should show for the overview.

Reply:
This was my brainfart 🎉